### PR TITLE
Feat : changes for 2.4.0 release

### PR DESCRIPTION
**Pull request by erezz-mov-ai**
##content
split of movai_classes

#71 

- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
'- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue'

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

**Review by engdorm — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -41,5 +42,11 @@
     classifiers=["Programming Language :: Python :: 3"],
     install_requires=requirements,
     data_files=data_files,
-    entry_points={},
+    entry_points={
+         "console_scripts":[
+             "backup = dal.tools.backup:main",
```
Comment (AlexFernandes-MOVAI) [inline setup.py:47]: might be better to add a prefix like : `movai_backup` or `dal_backup`

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
please add prefix for entrypoints

File: setup.py (hunk 1 above)
Comment (AlexFernandes-MOVAI) [reply setup.py:47]: easily testable with :
- `python3 setup.py build`
- `sudo python3 setup.py install`

there you will end up with a `backup` command:
```
cat /usr/local/bin/backup
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'dal==1.0.1.post0','console_scripts','backup'
__requires__ = 'dal==1.0.1.post0'
import re
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(
        load_entry_point('dal==1.0.1.post0', 'console_scripts', 'backup')()
    )
```

Comment (erezz-mov-ai) [reply setup.py]: done!

File: setup.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
     data_files=data_files,
     entry_points={
          "console_scripts":[
-             "backup = dal.tools.backup:main",
+             "dal_backup = dal.tools.da_backup:main",
```
Comment (engdorm) [inline setup.py]: dal.tools.da_backup -> dal.tools.dal_backup

**Review by engdorm — CHANGES_REQUESTED** [review]

File: setup.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "python-box==4.0.4",
     "redis==3.3.11",
     "yarl==1.7.2",
-    "movai_core_shared==1.0.1.0"
+    "movai_core_shared==2.4.0.1"
```
Comment (AlexFernandes-MOVAI) [inline setup.py:19]: let's use something like : `"movai_core_shared==2.4.*"`

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]

**Review by engdorm — APPROVED** [review]

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request closed without merging**